decidim/decidim
# Add Projects (Budgets) to filtered search

#### :tophat: What? Why?
Upon searching through modules as a user, Projects under Budgets were not searchable or available. 

#### :pushpin: Related Issues

- Fixes #11719

#### Testing
1. Head to Decidim 
2. Search by typing a Project name (or just by pressing 'Enter' while in the search)
3. See the searchable Projects in filters on the left.

RSPEC test added to ```decidim-core/spec/system/search_spec.rb``` also.

### :camera: Screenshots
Before: 

![image](https://github.com/decidim/decidim/assets/101816158/a8f6a918-ae47-4d92-9ebd-3301a193679d)

After: 

<img width="1431" alt="Screenshot 2023-10-11 at 12 13 14" src="https://github.com/decidim/decidim/assets/101816158/acfdf871-16dd-431c-8024-ea8db3eb0608">



:hearts: Thank you!
